tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -458,7 +458,7 @@ public static class Tags
         /// <summary>
         /// A CosmosDb container name.
         /// </summary>
-        internal const string CosmosDbContainer = "cosmosdb.container";
+        internal const string CosmosDbContainer = "messaging.destination";
```

Comment:
This change doesn't seem correct since this is a database span not a message queue span, but I don't know what the right tag name should be

Reply:
Yes I clearly need to fix it, thanks a lot.

Reply:
The change to `db.cosmosdb.container` looks good 🎉